slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -341,3 +356,23 @@ This is so we always have an accurate picture of RAP in the department to share
 :::
 
 # Final quiz!
+
+# Making time for learning
+
+It may feel difficult to work on your technical skills when you have fewer opportunities to do hands-on work. 
+
```

Comment:
could reference the L&D HoP knowledge share here as an almost mandatory thing they should all have watched.